fastapi/openapi/docs.py

```diff
@@ -34,6 +35,10 @@ def get_swagger_ui_html(
         url: '{openapi_url}',
     """
 
+    if swagger_ui_parameters:
+        for key, value in swagger_ui_parameters.items():
+            html += f"{key}: {value},"
```

Comment:
To follow the current JS rendering, you should add an `\n` at the end.
I don't think it's a good idea to pass JS code in argument `{value}`. It's easy to break all the javascript rendering in the browser. Why not using JSON to escape?
```
>>> json.dumps(False)
'false'
>>> json.dumps("string")
'"string"'
>>> json.dumps(1)
'1'
```

Reply:
Great point, I've made the changes and rebased onto master.